data/VT/incentive_relationships.json

```diff
@@ -1,17 +1,25 @@
 {
   "prerequisites": {
-    "VT-29": "VT-27",
-    "VT-30": "VT-28"
+    "VT-2": "VT-1",
+    "VT-4": "VT-3",
+    "VT-8": "VT-7",
+    "VT-10": { "anyOf": ["VT-7", "VT-18"] },
+    "VT-13": "VT-12",
+    "VT-21": "VT-18",
+    "VT-25": "VT-23",
+    "VT-38": "VT-37",
+    "VT-42": "VT-41",
+    "VT-44": "VT-43"
   },
   "exclusions": {
+    "VT-3": ["VT-1"],
+    "VT-10": ["VT-8", "VT-21"],
```

Comment:
Green Mountain Power's income bonus for ducted or ductless heat pumps excludes EVT's income bonuses for same.